crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -148,6 +148,7 @@ impl TransactionBlockEffects {
                         module_id,
                         source_line_number,
                         error_info,
+                        error_code: _,
```

Comment:
This should be included in the formatted clever error too, or do you want to do that in a separate PR?

Reply:
+1, we should include this in the error message.

We should also add a new field to the GraphQL effects structure that returns this abort code if it's a clever error, or otherwise returns the original abort code (if it's not a clever error). This is what would allow someone to always rely on a stable identifier.

For now, feel free to add it as an `abortCode` field or similar, next to the `errors` field. For the GraphQL Beta we can look at re-arranging things so they are better structured.